### PR TITLE
Cult Tweaks

### PR DESCRIPTION
**Pull request by DeityLink**
- Attempting to summon Nar-Sie elswhere than on the station will warn the cultists. If they don't abort their summoning they'll all get gibbed.
- Admins can now use the voice of Nar-Sie to talk to a single cultist, rather than to all of them (more immersive than using Subtle Messages)
- The Cult Armor rune will now cloth cultists in spaceworthy armor if the conditions for Nar-Sie's summoning have been completed or Nar-Sie has risen. (AT LAST)
- Tweaked the Cult Spacesuit sprite so its user no longer appears bare footed.
- Communicate runes now lets you automatically recognize the voice of whoever is using it. Telling your name with the rune at roundstart isn't mandatory anymore.
- Tweaked the Arcane Tome sprite.
- Added animations for the Summon Tome rune, and for when you copy words on a tome.
- Fixed non-cultists being able to copy words from an Arcane Tome to another one.
- You can now place items on Cult "tome" structures (such as the one in the librarian's closet) just like racks.
- Hell will only be initialized the first time Nar-Sie is spawned, so admins can spawn multiple Nar-Sies without causing multiple instances of Hell to overlap, with all the server-ending features it'd create.

### CODE REVIEW

**Comment by optimumtact** [comment]
Kicking Rad place to be

File: code/game/gamemodes/cult/cult_structures.dm [hunk 1]
```diff
@@ -74,7 +74,12 @@
 	icon_state = "tomealtar"
 	luminosity = 2
 	l_color = "#3e0000"
-//	luminosity = 5
+
+/obj/structure/cult/tome/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(isrobot(user))
```
Comment (ComicIronic) [inline code/game/gamemodes/cult/cult_structures.dm]: This is actually not necessary: drop_item already does nothing for robots.

**Comment by NigglyWiggly** [comment]
>Tweaked the Arcane Tome sprite.

T-thanks

**Comment by DeityLink** [comment]
@NigglyWiggly 
tome

**Comment by NigglyWiggly** [comment]
B-but muh stealthy tome!

In all seriousness my original version was probably way too dark

File: code/game/gamemodes/cult/cult_structures.dm (hunk 1 above)
Comment (DeityLink) [inline code/game/gamemodes/cult/cult_structures.dm]: oh, well, ok then, I might remove it from tables_racks.dm as well since I took the code from there.

**Comment by DeityLink** [comment]
Yes it was.

**Comment by DeityLink** [comment]
fixed conflicts and did two huge quality of life improvements:

* cultists can now cancel out of rune scribing, as well as see which words they previously picked (see pic)
update1

* cultists that examine a working rune will now know which spell it'll proc
update2

**Pull request merged**